web/src/main/java/org/springframework/security/web/savedrequest/SimpleSavedRequest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004, 2005, 2006 Acegi Technology Pty Limited
+ * Copyright 2004, 2005, 2006, 2020 Acegi Technology Pty Limited
```

Comment:
Thank you for updating this; however, the Acegi copyright headers are a funny exception to the rule for now. Please leave them as-is.

Reply:
Okay, I'll revert it.